app/views/upload.scala.html

```diff
@@ -38,11 +38,11 @@ <h2 class="govuk-error-summary__title logged-out-title" id="logged-out-error-tit
                 <div class="drag-and-drop">
                     <div class="govuk-summary-list">
                         <div class="govuk-summary-list__row">
-                            <dd id="drag-and-drop-success" class="govuk-summary-list__value drag-and-drop__success" hidden tabindex="-1" role="dialog" aria-describedby="successMessageText">
-			        <div>
+                            <dd id="drag-and-drop-success" class="govuk-summary-list__value drag-and-drop__success" hidden tabindex="-1" role="alert" aria-describedby="successMessageText">
```

Comment:
This was my question when I looked at it. I can only assume that the person who added it might have misread this part "[Dialogs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role) are generally placed on top of the rest of the page content" but I think this means, "literally" on top of the content like a dialog box.

Reply:
That was me I think and I can't remember why I did it other than vaguely thinking I read somewhere that you were supposed to do that but if you use alert then the screen reader goes "Notification...something something" which is what we're after so the change makes sense.